hnsecure/src/main/java/ca/bc/gov/hlth/hnsecure/filedrops/ResponseFileDropGenerater.java

```diff
@@ -25,12 +25,11 @@
 
 	@Handler
 	public void createFileDrops(Exchange exchange) {
-		String fileName = buildFileNameParameters(exchange,exchange.getIn().getMessageId());
+		String fileName = buildFileNameParameters(exchange,exchange.getExchangeId());
```

Comment:
Switched to using the ExchangeId instead of the message Id. 
For Camel Http they're essentially the same thing, but for other components that isn't always true.